capitals.rb

```diff
@@ -1,5 +1,7 @@
+module StateCapitals
+
 # an array of state hashes
```

Comment:
I'd recommend encapsulating the `states` in a method called something like `get_state_capitals_array` that simply returns the `states`. after using `require_relative` in your other file `caps_game.rb` you can say something like `states=get_state_capitals_array()` and get all the state